modules/dcache/src/main/java/org/dcache/pool/classic/IoQueueManager.java

```diff
@@ -451,16 +461,43 @@ public Serializable call() throws NoSuchElementException
                     comparator = comparator.reversed();
                 }
 
+                // Prepare filters
+                Predicate<MoverRequestScheduler.PrioritizedRequest> clientIpFilter;
+                if (clientHost != null && !clientHost.isEmpty()) {
```

Comment:
What if I say `mover ls --client ""`? Wouldn't this cause such a scenario? Why is queueName being checked then?